src/app/components/Feedback/Feedback.jsx

```diff
@@ -1,217 +1,48 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import FocusTrap from 'focus-trap-react';
-import axios from 'axios';
-import {
-  Button,
-  Heading,
-  HelperErrorText,
-  Label,
-  Link,
-  TextInput
-} from '@nypl/design-system-react-components';
+import React, { useContext, useState } from 'react'
+import axios from 'axios'
 
+import { FeedbackBoxContext } from '../../context/FeedbackContext'
 import { trackDiscovery } from '../../utils/utils';
-import appConfig from '../../data/appConfig';
+import appConfig from '../../data/appConfig'
 
-const initialFields = () => ({
-  email: '',
-  feedback: '',
-});
-
-class Feedback extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      showForm: false,
-      fields: initialFields(),
-      success: false,
-      commentInputError: false,
-    };
-
-    this.commentText = React.createRef();
-    this.emailInput = React.createRef();
-
-    this.onSubmitForm = this.onSubmitForm.bind(this);
-    this.toggleForm = this.toggleForm.bind(this);
-    this.deactivateForm = this.deactivateForm.bind(this);
-    this.handleInputChange = this.handleInputChange.bind(this);
-  }
-
-  onSubmitForm(e) {
-    e.preventDefault();
-    const { fields } = this.state;
-    if (!fields.feedback && !fields.feedback.length) {
-      this.setState({ commentInputError: true }, () => this.commentText.current.focus())
-    } else {
-      this.postForm();
-      trackDiscovery('Feedback', 'Submit');
-    }
+const Feedback = () => {
+  const [screen, setScreen] = useState('form')
+  const { FeedbackBox, isOpen, onClose, onOpen, itemMetadata, setItemMetadata } = useContext(FeedbackBoxContext)
+  const closeAndResetItemMetadata = () => {
+    if (itemMetadata) setItemMetadata(null)
+    trackDiscovery('Feedback', 'Close');
+    onClose()
   }
-
-  postForm() {
-    const { fields } = this.state;
-    axios({
-      method: 'POST',
-      url: `${appConfig.baseUrl}/api/feedback`,
-      data: {
-        fields,
-      },
-    }).then((res) => {
+  const submitFeedback = async (metadataAndComment) => {
+    trackDiscovery('Feedback', 'Submit')
+    try {
+      const res = await axios.post(`${appConfig.baseUrl}/api/feedback`,
+        metadataAndComment)
```

Comment:
I was having trouble testing this locally. It kept only returning the error screen. I think it's because you have to pass the data as
```
{ fields: metadataAndComment }
```
instead of 
```
metadataAndComment
```
as the direct object. The browser console throws the error `Request body missing `field` key'`. On line 27, you may want to log `e.response.data.error` (if there is a value) to have a more descriptive error log. Or maybe just `e.response`.
<img width="1203" alt="Screen Shot 2022-12-21 at 4 49 29 PM" src="https://user-images.githubusercontent.com/1280564/209008803-ff4dbced-6afb-40b0-acde-48f9bf0c3676.png">